Add open source report (#414)
ombulabs/blog: _posts/2020-Q1-open-source-report.markdown

---
layout: post
title: 2020-q1-open-source-report
date: 2020-04-09 20:12:12

> categories:[open source]

Reviewer: ```suggestion
categories: [open-source]
```

author: bronzdoc

---

> # OmbuLabs Open Source 2020 Q1 report

Reviewer: ```suggestion
# Open Source at OmbuLabs: 2020 Q1 Report
```

![ombulabs open source report](https://external-preview.redd.it/zbCp6e3VikD9vsKOtTGVKI90UnWIOZN5NMRNSvgvXxE.jpg?width=960&crop=smart&auto=webp&s=9318c01ae704565ad90f4736aec5af6b671d98fa)

# Relevant News

## Skunk

> [Skunk](https://github.com/fastruby/skunk) Is a RubyCritic extension to calculate SkunkScore for a file or project.

Reviewer: ```suggestion
[Skunk](https://github.com/fastruby/skunk) is a RubyCritic extension to calculate the [SkunkScore](https://www.fastruby.io/blog/code-quality/intruducing-skunk-stink-score-calculator.html) for a file or project.
```

From Jan 01, 2020 through Apr 05, 2020, fastruby/skunk gained 58 new commits, contributed by 5 authors. There were 506 additions and 269 deletions across 26 files.

### New Contributors
There were 2 new contributors during this report period.

[Grzegorz Jakubiak](https://github.com/grzegorz-jakubiak)

> [Sebastián Palma](https://github.com/sebastian-palma)

Reviewer: 💪

### Pull Requests
A total of 11 pull requests were opened during this period.

Average age: approximately 1 month and 3 weeks.
Oldest pull request was opened on Jan 30, 2020.
Newest pull request was opened on Feb 29, 2020.

### Issues
A total of 3 issues were opened during this period.

Average age: approximately 1 month and 3 weeks.
Oldest issue was opened on Jan 14, 2020.
Newest issue was opened on Mar 10, 2020.

### New Logo:
Skunk has a new logo and we also have stickers so make sure to ping [Ernesto](https://github.com/etagwerker) to give you some.

<div style="text-align: center; width: 500px;">
  <img src="/blog/assets/images/skunk_logo.png" alt="skunk">
</div>

## BundlerLeak

> [bundler-leak](https://github.com/rubymem/bundler-leak) Is a tool to find leaky gems in your dependencies.

Reviewer: ```suggestion
[bundler-leak](https://github.com/rubymem/bundler-leak) is a tool to find leaky gems in your dependencies.
```

From Jan 01, 2020 through Apr 02, 2020, rubymem/bundler-leak gained 15 new commits, contributed by 4 authors. There were 148 additions and 76 deletions across 11 files.

### New Contributors
There were 2 new contributors during this report period.

[Alexander Popov](https://github.com/AlexWayfer)

> [chubchenko](https://github.com/chubchenko)

Reviewer: 🎉

### Pull Requests
A total of 5 pull requests were opened during this period.

Average age: approximately 1 month and 3 weeks.
Oldest pull request was opened on Jan 11, 2020.
Newest pull request was opened on Feb 19, 2020.

## RubyMem.com

> [RubyMem.com](https://github.com/rubymem/rubymem.com) Is the website to report leaky gems.

Reviewer: ```suggestion
[RubyMem.com](https://github.com/rubymem/rubymem.com) is the website to report leaky gems.
```

From Jan 01, 2020 through Apr 05, 2020, rubymem/rubymem.com gained 3 new commits, contributed by 2 authors. There were 3 additions and 3 deletions across 1 file.

### New Contributors
There were no new contributors during this report period.

### Pull Requests
A total of 2 pull requests were opened during this period.

Average age: approximately 1 month and 6 days.
Oldest pull request was opened on Feb 26, 2020.
Newest pull request was opened on Mar 03, 2020.

## BlogCop

[Blogcop](https://github.com/ombulabs/blogcop) is a simple GitHub bot made with Sinatra that helps you manage the outdated articles of your Jekyll blog.

From Jan 01, 2020 through Apr 05, 2020, ombulabs/blogcop gained 1 new commit, contributed by 1 author. There was 1 addition and 1 deletion across 1 file.

### New Contributors
There was 1 new contributor during this report period.

[Luis Sagastume](https://github.com/bronzdoc)

### Issues
A total of 1 issue was opened during this period.

Average age: approximately 1 month and 3 weeks.
Oldest issue was opened on Feb 12, 2020.
Newest issue was opened on Feb 12, 2020.

> ### Outside contributions by our team members

Reviewer: ```suggestion
### Outside Contributions by Our Team Members
```

> At OmbuLabs we can dedicate a certain ammount of time to contribute to external open source projects, these are some of them that we have contributed in the past months

Reviewer: ```suggestion
At OmbuLabs we can dedicate a certain amount of time to contribute to external open source projects, these are some of the projects that we have contributed to in the past months
```

---
layout: post

> title: 2020-q1-open-source-report

Reviewer 2: @bronzdoc Here you should put the real title of the article, and with quotes (otherwise raises a syntax error)

date: 2020-04-09 20:12:12
categories: [open-source]
author: bronzdoc

---

> 

Reviewer 2: @bronzdoc I think this article needs an introduction. This is the first report that we do, so no one is familiar with it.

Reviewer 2: @bronzdoc also, you're missing the `<!--more-->` tag. It shows the entire article in the index page without it. We don't want to have the image before that tag.

# Open Source at OmbuLabs: 2020 Q1 Report

> ![ombulabs open source report](https://external-preview.redd.it/zbCp6e3VikD9vsKOtTGVKI90UnWIOZN5NMRNSvgvXxE.jpg?width=960&crop=smart&auto=webp&s=9318c01ae704565ad90f4736aec5af6b671d98fa)

Reviewer 2: @bronzdoc Please download this image and put it in the assets. Also, this should be an html so we can have class and alt properties. for example:

`<img src="/assets/images/team_retreat/team-retreat-official.jpg" alt="team retreat ombulabs" class="medium-img">`

# Relevant News

## Skunk

[Skunk](https://github.com/fastruby/skunk) is a RubyCritic extension to calculate the [SkunkScore](https://www.fastruby.io/blog/code-quality/intruducing-skunk-stink-score-calculator.html) for a file or project.

> From Jan 01, 2020 through Apr 05, 2020, fastruby/skunk gained 58 new commits, contributed by 5 authors. There were 506 additions and 269 deletions across 26 files.

Reviewer 2: @bronzdoc I find this a bit hard to read. Can you make some refactor to it?

Author: 🤔 This is auto generated by the reporting tool. I would prefer not to change it

### New Contributors
There were 2 new contributors during this report period.

[Grzegorz Jakubiak](https://github.com/grzegorz-jakubiak)

[Sebastián Palma](https://github.com/sebastian-palma)

### Pull Requests
A total of 11 pull requests were opened during this period.

Average age: approximately 1 month and 3 weeks.
Oldest pull request was opened on Jan 30, 2020.
Newest pull request was opened on Feb 29, 2020.

### Issues
A total of 3 issues were opened during this period.

Average age: approximately 1 month and 3 weeks.
Oldest issue was opened on Jan 14, 2020.
Newest issue was opened on Mar 10, 2020.

### New Logo:
Skunk has a new logo and we also have stickers so make sure to ping [Ernesto](https://github.com/etagwerker) to give you some.

<div style="text-align: center; width: 500px;">
  <img src="/blog/assets/images/skunk_logo.png" alt="skunk">
</div>

## BundlerLeak

[bundler-leak](https://github.com/rubymem/bundler-leak) is a tool to find leaky gems in your dependencies.

> From Jan 01, 2020 through Apr 02, 2020, rubymem/bundler-leak gained 15 new commits, contributed by 4 authors. There were 148 additions and 76 deletions across 11 files.

Reviewer 2: @bronzdoc Same as above. Not very easy to read

### New Contributors
There was 1 new contributor during this report period.

[Luis Sagastume](https://github.com/bronzdoc)

### Issues
A total of 1 issue was opened during this period.

Average age: approximately 1 month and 3 weeks.
Oldest issue was opened on Feb 12, 2020.
Newest issue was opened on Feb 12, 2020.

### Outside Contributions by Our Team Members

At OmbuLabs we can dedicate a certain amount of time to contribute to external open source projects, these are some of the projects that we have contributed to in the past months

### Ernesto ([etagwerker](https://github.com/etagwerker))

#### Pull Requests

| Project | Pull Request |
| --- | ----------- |
| `attr-encrypted/attr_encrypted` | [Stop testing EOLed ruby versions in Travis](https://github.com/attr-encrypted/attr_encrypted/pull/364) |
| `rails/rails` | [Grammar update](https://github.com/rails/rails/pull/38441)
|  `devise-security/devise-security` | [Fixes/documentation for password complexity validator](https://github.com/devise-security/devise-security/pull/179) |
---

> ### Fiona ([FionaDl](https://github.com/FionaDL))

Reviewer 2: ```suggestion
### Fiona ([FionaDL](https://github.com/FionaDL))
```

---
layout: post

> title: "2020 Q1 open-source-report"

Reviewer: @bronzdoc This continues to look a little weird. Why not just use a title like this? 

```suggestion
title: "Open Source Report (2020 Q1)"
```

> date: 2020-04-09 20:12:12

Reviewer: @bronzdoc You'll need to update the date here when all comments have been addressed.
